src/render.ts

```diff
@@ -115,7 +115,7 @@ export async function render(
     }
     case "Vue": {
       const Vue = await require("vue");
-      const app = storyResult.app;
+      const app = storyResult.app || (div as any).__vue_app__;
```

Comment:
IIUC `__vue_app__` is Vue 3 specific, but so is the rest of the Vue logic in this case block
if that is so, I guess we don't support Vue 2 here and I don't need to test is with it, right?